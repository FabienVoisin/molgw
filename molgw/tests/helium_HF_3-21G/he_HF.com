#p HF/3-21G scf=tight

he

0  1
He  0.000   0.000   0.000

