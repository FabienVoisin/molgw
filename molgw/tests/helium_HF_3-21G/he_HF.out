 
 Welcome to the fascinating world of ATOM
 
 

 WARNING: OPENMP option is activated with threads number     1
 before read
 inside read_input

 WARNING: calculation name: HF

 Summary of the input parameters 

  calculation type:   HF
             Natom:   1
         Electrons:   2.0000
            Charge:   0.0000
     Magnetization:   0.0000
         Basis set:   3-21G                                                                                               
 Spin polarization:   1
         SCF steps:  20
            Mixing:   0.8000
 
 
 Element used for Z value:    He
 Element used for the basis:  He
 Basis type: 3-21G

 WARNING: basis file used: He_3-21G
 
 open the basis set file He_3-21G
 Total number of basis functions           2
 
 Basis set is ready and fit

 Allocating the ERI array:    0.000 [Mb]
 success

 Calculate and store all the Electron Repulsion Integrals (ERI)
 number of shells           3
 integral (ss|ss)  6.380253779808154E-003
 integral (ss|ss)  1.254614129465390E-002
 integral (ss|ss)  1.426508878710658E-002
 integral (ss|ss)  1.254614129465390E-002
 integral (ss|ss)  5.742824726675862E-002
 integral (ss|ss)  9.898554700946324E-002
 integral (ss|ss)  1.426508878710658E-002
 integral (ss|ss)  9.898554700946324E-002
 integral (ss|ss)  0.316616721685354     
 integral (ss|ss)  1.254614129465390E-002
 integral (ss|ss)  2.563106520744702E-002
 integral (ss|ss)  2.935749119388816E-002
 integral (ss|ss)  2.563106520744702E-002
 integral (ss|ss)  0.126396363325277     
 integral (ss|ss)  0.221469384305618     
 integral (ss|ss)  2.935749119388816E-002
 integral (ss|ss)  0.221469384305618     
 integral (ss|ss)  0.721968394791857     
 integral (ss|ss)  1.426508878710658E-002
 integral (ss|ss)  2.935749119388816E-002
 integral (ss|ss)  3.367581641364768E-002
 integral (ss|ss)  2.935749119388816E-002
 integral (ss|ss)  0.147170158061256     
 integral (ss|ss)  0.258913872090408     
 integral (ss|ss)  3.367581641364768E-002
 integral (ss|ss)  0.258913872090408     
 integral (ss|ss)  0.848167257554735     
 integral (ss|ss)  1.254614129465390E-002
 integral (ss|ss)  2.563106520744702E-002
 integral (ss|ss)  2.935749119388816E-002
 integral (ss|ss)  2.563106520744702E-002
 integral (ss|ss)  0.126396363325277     
 integral (ss|ss)  0.221469384305618     
 integral (ss|ss)  2.935749119388816E-002
 integral (ss|ss)  0.221469384305618     
 integral (ss|ss)  0.721968394791857     
 integral (ss|ss)  5.742824726675862E-002
 integral (ss|ss)  0.126396363325277     
 integral (ss|ss)  0.147170158061256     
 integral (ss|ss)  0.126396363325277     
 integral (ss|ss)  0.773735677360008     
 integral (ss|ss)   1.45390399374462     
 integral (ss|ss)  0.147170158061256     
 integral (ss|ss)   1.45390399374462     
 integral (ss|ss)   5.23296560247579     
 integral (ss|ss)  9.898554700946323E-002
 integral (ss|ss)  0.221469384305618     
 integral (ss|ss)  0.258913872090408     
 integral (ss|ss)  0.221469384305618     
 integral (ss|ss)   1.45390399374462     
 integral (ss|ss)   2.82409274804356     
 integral (ss|ss)  0.258913872090408     
 integral (ss|ss)   2.82409274804356     
 integral (ss|ss)   10.8053664711890     
 integral (ss|ss)  1.426508878710658E-002
 integral (ss|ss)  2.935749119388816E-002
 integral (ss|ss)  3.367581641364768E-002
 integral (ss|ss)  2.935749119388816E-002
 integral (ss|ss)  0.147170158061256     
 integral (ss|ss)  0.258913872090408     
 integral (ss|ss)  3.367581641364768E-002
 integral (ss|ss)  0.258913872090408     
 integral (ss|ss)  0.848167257554735     
 integral (ss|ss)  9.898554700946323E-002
 integral (ss|ss)  0.221469384305618     
 integral (ss|ss)  0.258913872090408     
 integral (ss|ss)  0.221469384305618     
 integral (ss|ss)   1.45390399374462     
 integral (ss|ss)   2.82409274804356     
 integral (ss|ss)  0.258913872090408     
 integral (ss|ss)   2.82409274804356     
 integral (ss|ss)   10.8053664711890     
 integral (ss|ss)  0.316616721685354     
 integral (ss|ss)  0.721968394791857     
 integral (ss|ss)  0.848167257554735     
 integral (ss|ss)  0.721968394791857     
 integral (ss|ss)   5.23296560247579     
 integral (ss|ss)   10.8053664711890     
 integral (ss|ss)  0.848167257554735     
 integral (ss|ss)   10.8053664711890     
 integral (ss|ss)   48.1767611189574     
 Done.
 

 === occupations ===
   1      2.00000
   2      0.00000
 

-------------------------------------------
 *** SCF cycle No:    1

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.43185    -11.75112
   2      2.19629     59.76416
 
 convergence criterium on the density matrix   1.91220210790658     
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     7.8322590537
    Nucleus Energy  [Ha]:   -10.9793930077
    Hartree Energy  [Ha]:     3.6065197351
    Exchange Energy [Ha]:    -1.8032598676
 
    Total Energy    [Ha]:    -1.3438740865
 

-------------------------------------------
 *** SCF cycle No:    2

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.79261    -21.56802
   2      2.12070     57.70729
 
 convergence criterium on the density matrix  0.447065518049926     
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     4.0209203127
    Nucleus Energy  [Ha]:    -7.7697073734
    Hartree Energy  [Ha]:     2.3677387673
    Exchange Energy [Ha]:    -1.1258727013
 
    Total Energy    [Ha]:    -2.5069209948
 

-------------------------------------------
 *** SCF cycle No:    3

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.87764    -23.88189
   2      2.09342     56.96498
 
 convergence criterium on the density matrix  0.105267551672976     
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     3.1154369625
    Nucleus Energy  [Ha]:    -6.9637495696
    Hartree Energy  [Ha]:     2.1299304055
    Exchange Energy [Ha]:    -1.0499847759
 
    Total Energy    [Ha]:    -2.7683669775
 

-------------------------------------------
 *** SCF cycle No:    4

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.89747    -24.42128
   2      2.08503     56.73663
 
 convergence criterium on the density matrix  2.524322873894036E-002
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.9022392110
    Nucleus Energy  [Ha]:    -6.7650983369
    Hartree Energy  [Ha]:     2.0750662971
    Exchange Energy [Ha]:    -1.0343412387
 
    Total Energy    [Ha]:    -2.8221340675
 

-------------------------------------------
 *** SCF cycle No:    5

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90212    -24.54792
   2      2.08262     56.67092
 
 convergence criterium on the density matrix  6.153950972618855E-003
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8516871695
    Nucleus Energy  [Ha]:    -6.7160937219
    Hartree Energy  [Ha]:     2.0615810381
    Exchange Energy [Ha]:    -1.0301405582
 
    Total Energy    [Ha]:    -2.8329660725
 

-------------------------------------------
 *** SCF cycle No:    6

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90322    -24.57795
   2      2.08195     56.65272
 
 convergence criterium on the density matrix  1.518900023978123E-003
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8395887029
    Nucleus Energy  [Ha]:    -6.7039599788
    Hartree Energy  [Ha]:     2.0582074810
    Exchange Energy [Ha]:    -1.0289730495
 
    Total Energy    [Ha]:    -2.8351368444
 

-------------------------------------------
 *** SCF cycle No:    7

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90349    -24.58514
   2      2.08177     56.64780
 
 convergence criterium on the density matrix  3.780971024867868E-004
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8366674979
    Nucleus Energy  [Ha]:    -6.7009445839
    Hartree Energy  [Ha]:     2.0573593065
    Exchange Energy [Ha]:    -1.0286534722
 
    Total Energy    [Ha]:    -2.8355712517
 

-------------------------------------------
 *** SCF cycle No:    8

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90355    -24.58688
   2      2.08172     56.64650
 
 convergence criterium on the density matrix  9.464695912978111E-005
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8359566377
    Nucleus Energy  [Ha]:    -6.7001928759
    Hartree Energy  [Ha]:     2.0571457022
    Exchange Energy [Ha]:    -1.0285676122
 
    Total Energy    [Ha]:    -2.8356581483
 

-------------------------------------------
 *** SCF cycle No:    9

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58730
   2      2.08171     56.64616
 
 convergence criterium on the density matrix  2.377689045388589E-005
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357824916
    Nucleus Energy  [Ha]:    -6.7000050017
    Hartree Energy  [Ha]:     2.0570918673
    Exchange Energy [Ha]:    -1.0285448857
 
    Total Energy    [Ha]:    -2.8356755285
 

-------------------------------------------
 *** SCF cycle No:   10

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58741
   2      2.08170     56.64607
 
 convergence criterium on the density matrix  5.986366114835122E-006
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357395880
    Nucleus Energy  [Ha]:    -6.6999579487
    Hartree Energy  [Ha]:     2.0570782930
    Exchange Energy [Ha]:    -1.0285389369
 
    Total Energy    [Ha]:    -2.8356790046
 

-------------------------------------------
 *** SCF cycle No:   11

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58743
   2      2.08170     56.64604
 
 convergence criterium on the density matrix  1.509231618139312E-006
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357289683
    Nucleus Energy  [Ha]:    -6.6999461446
    Hartree Energy  [Ha]:     2.0570748692
    Exchange Energy [Ha]:    -1.0285373927
 
    Total Energy    [Ha]:    -2.8356796998
 

-------------------------------------------
 *** SCF cycle No:   12

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58744
   2      2.08170     56.64604
 
 convergence criterium on the density matrix  3.808002552183199E-007
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357263294
    Nucleus Energy  [Ha]:    -6.6999431793
    Hartree Energy  [Ha]:     2.0570740054
    Exchange Energy [Ha]:    -1.0285369943
 
    Total Energy    [Ha]:    -2.8356798389
 

-------------------------------------------
 *** SCF cycle No:   13

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58744
   2      2.08170     56.64604
 
 convergence criterium on the density matrix  9.612619547699426E-008
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357256716
    Nucleus Energy  [Ha]:    -6.6999424336
    Hartree Energy  [Ha]:     2.0570737874
    Exchange Energy [Ha]:    -1.0285368920
 
    Total Energy    [Ha]:    -2.8356798667
 

-------------------------------------------
 *** SCF cycle No:   14

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58744
   2      2.08170     56.64604
 
 convergence criterium on the density matrix  2.427174024897699E-008
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357255072
    Nucleus Energy  [Ha]:    -6.6999422459
    Hartree Energy  [Ha]:     2.0570737324
    Exchange Energy [Ha]:    -1.0285368659
 
    Total Energy    [Ha]:    -2.8356798723
 

-------------------------------------------
 *** SCF cycle No:   15

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58744
   2      2.08170     56.64604
 
 convergence criterium on the density matrix  6.129452355406150E-009
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357254660
    Nucleus Energy  [Ha]:    -6.6999421987
    Hartree Energy  [Ha]:     2.0570737185
    Exchange Energy [Ha]:    -1.0285368592
 
    Total Energy    [Ha]:    -2.8356798734
 

-------------------------------------------
 *** SCF cycle No:   16

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58744
   2      2.08170     56.64604
 
 convergence criterium on the density matrix  1.548006972242796E-009
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357254557
    Nucleus Energy  [Ha]:    -6.6999421867
    Hartree Energy  [Ha]:     2.0570737150
    Exchange Energy [Ha]:    -1.0285368575
 
    Total Energy    [Ha]:    -2.8356798736
 

-------------------------------------------
 *** SCF cycle No:   17

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58744
   2      2.08170     56.64604
 
 convergence criterium on the density matrix  3.909640313070495E-010
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357254531
    Nucleus Energy  [Ha]:    -6.6999421837
    Hartree Energy  [Ha]:     2.0570737141
    Exchange Energy [Ha]:    -1.0285368570
 
    Total Energy    [Ha]:    -2.8356798736
 

-------------------------------------------
 *** SCF cycle No:   18

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58744
   2      2.08170     56.64604
 
 convergence criterium on the density matrix  9.874194214034988E-011
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357254524
    Nucleus Energy  [Ha]:    -6.6999421830
    Hartree Energy  [Ha]:     2.0570737139
    Exchange Energy [Ha]:    -1.0285368569
 
    Total Energy    [Ha]:    -2.8356798736
 

-------------------------------------------
 *** SCF cycle No:   19

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58744
   2      2.08170     56.64604
 
 convergence criterium on the density matrix  2.493825580713199E-011
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357254522
    Nucleus Energy  [Ha]:    -6.6999421828
    Hartree Energy  [Ha]:     2.0570737138
    Exchange Energy [Ha]:    -1.0285368569
 
    Total Energy    [Ha]:    -2.8356798736
 

-------------------------------------------
 *** SCF cycle No:   20

 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -0.90357    -24.58744
   2      2.08170     56.64604
 
 convergence criterium on the density matrix  6.298691643713124E-012
 
    Nucleus-Nucleus [Ha]:     0.0000000000
    Kinetic Energy  [Ha]:     2.8357254522
    Nucleus Energy  [Ha]:    -6.6999421827
    Hartree Energy  [Ha]:     2.0570737138
    Exchange Energy [Ha]:    -1.0285368569
 
    Total Energy    [Ha]:    -2.8356798736
 
 ==================================================
 The SCF loop ends here
 ==================================================
 

                 --- Timings in (s) and # of calls ---
                    Total time          0.01
 
         4 gaussians integrals          0.00
 
                       Hartree          0.00        20
                      Exchange          0.00        20
                        DFT xc          0.00         0
 
           ERI basis transform          0.00         0
 
         Total GW polarization          0.00         0
            Diago 2 particle H          0.00         0
           Invert 2 particle S          0.00         0
 
                GW self-energy          0.00         0
 
                    MP2 energy          0.00         0
               MP2 self-energy          0.00         0
                 ----------------------

 SUMMARY of all the WARNINGS issued along the run:

 1.     OPENMP option is activated with threads number     1                                                
 2.     calculation name: HF                                                                                
 3.     basis file used: He_3-21G                                                                           



 This is the end

